lib/private/Files/Storage/Local.php

```diff
@@ -399,8 +399,15 @@ public function fopen($path, $mode) {
 		return $result;
 	}
 
-	public function hash($type, $path, $raw = false) {
-		return hash_file($type, $this->getSourcePath($path), $raw);
+	public function hash($type, $path, $raw = false): string|false {
+		/** @var string|false|null */
+		$hash = hash_file($type, $this->getSourcePath($path), $raw);
+
+		if ($hash === null) {
+			return false;
+		}
+
+		return $hash;
```

Comment:
```suggestion
		return hash_file($type, $this->getSourcePath($path), $raw);
```

To apply Come's suggestion